clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -587,6 +587,8 @@ check_function_markers()
     constexpr memref_tid_t TID = 1;
```

Comment:
I would mention tailcalls in the PR description as well.

Reply:
Done.